MUI/src/main/java/mui/MUILogProvider.java

```diff
@@ -44,7 +47,7 @@ public void runMUI(String manticoreExePath, String programPath,
 		MUILogContentComponent newTabContent = new MUILogContentComponent();
 
 		newTabContent.MUIInstance
-				.callProc(buildCommand(manticoreExePath, programPath, formOptions, moreArgs));
+				.callProc(buildCommand("manticore", programPath, formOptions, moreArgs), defPort);
```

Comment:
this will be fixed/changed in a PR for https://github.com/trailofbits/MUI-Ghidra/issues/6 that I'll open in the next 24h